tests/excel/test_basic_excel.py

```
@@ -0,0 +1,115 @@
+"""Excel対応の基本テスト."""
+
+import shutil
+import tempfile
+from pathlib import Path
+
+import pytest
+
+
+# Excel対応の基本テスト（ImportErrorは内部で処理）
```

Ruff (0.11.9)

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)